client/analytics/components/report-table/index.js

```diff
@@ -317,8 +317,9 @@ class ReportTable extends Component {
 		const totals = get( primaryData, [ 'data', 'totals' ], {} );
 		const totalResults = items.totalResults;
 		const downloadable = 0 < totalResults;
-		const searchWords = getSearchWords( query );
-		const searchedLabels = searchWords.map( v => ( { id: v, label: v } ) );
+		// Search words are in the query string, not the table query.
+		const searchWords = getSearchWords( this.props.query );
```

Comment:
Destructuring `query` from `this.props` above would be cleaner.

Reply:
I didn't go that route because we're destructuring `query` from `tableData` on line 308.